DOC-576-change-logical-partitions-to-pre-replication (#70)
redpanda-data/cloud-docs: modules/reference/partials/tiers.adoc

When you create a BYOC cluster, you select your usage tier. Each tier provides tested workload configurations for maximum throughput, partitions, and connections. 

|=== 

> | Tier | Ingress | Egress | Partitions (pre-replication) | Connections

Reviewer: Is there a way to put a line break in the Partitions (pre-replication) heading cell? The break in the output is a bit unfortunate/distracting. Or, perhaps footnote it and put the pre-replication caveat there?

====
* On Azure, tiers 1-3 are supported. 
* Depending on the workload, it may not be possible to achieve all maximum values. For example, a high number of partitions may make it more difficult to reach the maximum value in throughput.

> * Partition counts assume the Redpanda Cloud default replication factor of 3. If you set a higher replication factor, the maximum value for partitions will be lower.

Reviewer: ```suggestion
* Partition counts are based on the assumption that the Redpanda Cloud default replication factor is 3. If you set a higher replication factor, the maximum value for partitions will be lower.
```

====
* On Azure, tiers 1-3 are supported. 
* Depending on the workload, it may not be possible to achieve all maximum values. For example, a high number of partitions may make it more difficult to reach the maximum value in throughput.

> * Partition counts assume the Redpanda Cloud default replication factor of 3. If you set a higher replication factor, the maximum value for partitions will be lower.

Reviewer: ```suggestion
* Partition counts are based on the assumption that the Redpanda Cloud default replication factor is 3. If you set a higher replication factor, the maximum value for partitions will be lower.
```
